content/influxdb/cloud-iox/query-data/sql/aggregate-select.md

```diff
@@ -0,0 +1,268 @@
+
+- The first is the column to operate on.
+- The second is the time column to use in the selection logic.
+
+In your `SELECT` statement, execute a selector function and use bracket notation
+to reference properties of the [returned struct](#selector-struct-schema) to
+populate the column value:
+
+```sql
+SELECT
+  selector_first(temp, time)['time'] AS time,
+  selector_first(temp, time)['value'] AS temp,
+  room
+FROM home
+GROUP BY room
+```
+
+## Example aggregate queries
+
+- [Performed an ungrouped aggregation](#performed-an-ungrouped-aggregation)
+- [Group and aggregate data](#group-and-aggregate-data)
+  - [Downsample data by applying interval-based aggregates](#downsample-data-by-applying-interval-based-aggregates)
+- [Query rows based on aggregate values](#query-rows-based-on-aggregate-values)
+
+{{% note %}}
+#### Sample data
+
+The following examples use the sample data written in the
+[Get started writing data guide](/influxdb/cloud-iox/get-started/write/).
+To run the example queries and return results,
+[write the sample data](/influxdb/cloud-iox/get-started/write/#write-line-protocol-to-influxdb)
+to your InfluxDB Cloud bucket before running the example queries.
+{{% /note %}}
+
+### Perform an ungrouped aggregation
+
+To aggregate _all_ queried values in a specified column:
+
+- Use aggregate or selector functions in your `SELECT` statement
+- Do not include a `GROUP BY` clause to leave your data ungrouped
+
+```sql
+SELECT avg(co) AS 'average co' from home
+```
+
+{{< expand-wrapper >}}
+{{% expand "View example results" %}}
+|    average co     |
+| :---------------: |
+| 5.269230769230769 |
+{{% /expand %}}
+{{< /expand-wrapper >}}
+
+### Group and aggregate data
+
+To apply aggregate or selector functions to data grouped:
+
+- Use aggregate or selector functions in your `SELECT` statement
+- Include columns to group by in your `SELECT` statement
+- Include a `GROUP BY` clause with a comma-delimited list of columns to group by
+
+```sql
+SELECT
+  room,
+  avg(temp) AS 'average temp'
+FROM home
+GROUP BY room
+```
+
+{{< expand-wrapper >}}
+{{% expand "View example results" %}}
+| room        |       average temp |
+| :---------- | -----------------: |
+| Living Room |  22.16923076923077 |
+| Kitchen     | 22.623076923076926 |
+{{% /expand %}}
+{{< /expand-wrapper >}}
+
+#### Downsample data by applying interval-based aggregates
+
+A common use case when querying time series is downsampling data by applying
+aggregates to time-based groups. To group and aggregate data into time-based
+groups:
+
+- In your `SELECT` clause:
+
+  - Use `DATE_BIN` to calculate windows of time based on a specified interval
+    and update the timestamp in the `time` column based on the start
+    boundary of the window that the original timestamp is in.    
+    For example, if you use `DATE_BIN` to window data into one day intervals,
+    {{% influxdb/custom-timestamps-span %}}`2022-01-01T12:34:56Z`{{% /influxdb/custom-timestamps-span %}}
```

Comment:
Maybe explicitly say that if 2022-01-01T12:34:56Z was your start time then your first date bin would start at 2022-01-01T12:34:56Z?